emc/estimator.py

```diff
@@ -0,0 +1,52 @@
+"""A SHORELine-like algorithm for the realignment of dMRI data."""
+import attr
+import numpy as np
+from pathlib import Path
+
+
+@attr.s(slots=True)
+class EddyMotionEstimation:
+    """Estimate rigid-body head-motion and distortions derived from eddy-currents."""
+
+    datapath = attr.ib(default=None)
+    """Path to the input DWI dataset."""
+    gradients = attr.ib(default=None)
+    """A 2D numpy array of the gradient table in RAS+B format."""
+    bzero = attr.ib(default=None)
+    """A *b=0* reference map, preferably obtained by some smart averaging."""
```

Comment:
Is this a path or an array? I would advocate for either.

Reply:
(numpy array)